tests/vm_test.go

```diff
@@ -804,12 +806,13 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Expect(err).ToNot(HaveOccurred())
 					Expect(newVM.Spec.RunStrategy).ToNot(BeNil())
 					Expect(*newVM.Spec.RunStrategy).To(Equal(v1.RunStrategyAlways))
+					By("Ensuring stateChangeRequests list is cleared")
 					Expect(len(newVM.Status.StateChangeRequests)).To(Equal(0))
 				})
 			})
 
 			Context("Using RunStrategyManual", func() {
-				It("should start", func() {
+				It("[test_id:2036] should start", func() {
```

Comment:
I am missing the action of start VM, and wait for VMI to start verify VM's runStrategy = Manual verify vm.status.stateChangeRequests is not present in vm's manifest

Reply:
Confirmed test is correct in offline conversation with @ILpinto